packages/opentelemetry-exporter-collector/src/platform/browser/CollectorExporter.ts

```diff
@@ -32,6 +37,22 @@ const DEFAULT_COLLECTOR_URL = 'http://localhost:55678/v1/trace';
 export class CollectorExporter extends CollectorExporterBase<
   CollectorExporterConfig
 > {
+  DEFAULT_HEADERS: { [key: string]: string } = {
+    [collectorTypes.OT_REQUEST_HEADER]: '1',
+  };
+  private _headers: { [key: string]: string };
+  private _useXHR: boolean = false;
+
+  /**
+   * @param config
+   */
+  constructor(config: CollectorExporterConfig = {}) {
+    super(config);
+    this._headers = config.headers || this.DEFAULT_HEADERS;
+    this._useXHR =
+      !!config.headers || typeof navigator.sendBeacon !== 'function';
```

Comment:
should this be `!!this._headers || ...`, or do we explicitly not care about sending only `DEFAULT_HEADERS` here?

Reply:
If there are custom headers to be sent, then spans must be sent using an XMLHTTPRequest, the beacon doesn't support arbitrary headers. If there are not custom headers, and the browser has beacon support, spans will be sent using it.

There is one default header, it's `collectorTypes.OT_REQUEST_HEADER` which is a signal to the XMLHTTPRequest plugin not to trace the request. This would come into play if there are not any custom headers, and the browser does not have beacon support. A better long term solution for this would be #1040. It would clean this up and we could remove default headers altogether.

Reply:
makes sense thanks!